android/build.gradle

```diff
@@ -5,21 +5,27 @@ import org.gradle.util.VersionNumber
 // sub-projects/modules.
 
 buildscript {
+    ext {
+        kotlinVersion = '1.6.10'
+    }
     repositories {
         google()
         jcenter()
     }
     dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
         classpath 'com.android.tools.build:gradle:4.1.2'
         classpath 'com.google.gms:google-services:4.3.4'
         classpath 'com.google.firebase:firebase-crashlytics-gradle:2.4.1'
+
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10"
```

Comment:
This dependency is duplicated. Also, is it necessary?

Reply:
Sorry, will remove duplication.